packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx

```diff
@@ -2,48 +2,54 @@ import { useState } from "react";
 import { Controller, useForm } from "react-hook-form";
```

Comment:
All diffs in this page are because the v1 version had the same name, and I moved v2 to the same place. So this is actually the difference between v1 and v2.